PubSub/src/Subscription.php

```diff
@@ -351,6 +351,36 @@ public function detached(array $options = [])
      *           be between 0 and 600 seconds. Defaults to 600 seconds.
      *     @type bool $enableExactlyOnceDelivery Indicates whether to enable
      *           'Exactly Once Delivery' on the subscription.
+     *     @type array cloudStorageConfig If delivery to Google Cloud Storage is used
+     *           with this subscription, this field is used to configure it.
+     *     @type string $cloudStorageConfig.bucket User-provided name for the Cloud Storage bucket.
+     *           The bucket must be created by the user. The bucket name must be without
+     *           any prefix like "gs://". See the [bucket naming
+     *           requirements] (https://cloud.google.com/storage/docs/buckets#naming).
+     *     @type string $cloudStorageConfig.filenamePrefix
+     *           User-provided prefix for Cloud Storage filename. See the [object naming
+     *           requirements](https://cloud.google.com/storage/docs/objects#naming).
+     *     @type string $cloudStorageConfig.filenameSuffix
+     *           User-provided suffix for Cloud Storage filename. See the [object naming
+     *           requirements](https://cloud.google.com/storage/docs/objects#naming). Must
+     *           not end in "/".
+     *     @type array $cloudStorageConfig.textConfig If set, message data will be written
+     *           to Cloud Storage in text format.
+     *     @type array $cloudStorageConfig.avroConfig If set, message data will be written to
+     *           Cloud Storage in Avro format.
+     *     @type bool $cloudStorageConfig.avroConfig.writeMetadata
+     *           When true, write the subscription name, message_id, publish_time,
+     *           attributes, and ordering_key as additional fields in the output.
+     *     @type Duration|string $cloudStorageConfig.maxDuration The maximum duration
```

Comment:
Why do we need both `Duration|string`?

Reply:
Other duration apis accepts both Duration and string. Eg: 
https://github.com/googleapis/google-cloud-php/blob/ea5dec2c3bf249300d2e19d87a10195231cf8534/PubSub/src/Subscription.php#L293

https://github.com/googleapis/google-cloud-php/blob/ea5dec2c3bf249300d2e19d87a10195231cf8534/PubSub/src/Subscription.php#L308